### ISSUE TITLE
Website patch #1  ::  update homepage messaging & copy

### ISSUE THREAD

**Issue by pmoorman** [rchain/bounties#522]
### Context

The rchain.coop main website has been an eyesore ever since I joined RChain, and many people share the sentiment. Messaging is also hopelessly outdated.

We have formed a 'Branding WG' (working group) to work on branding, messaging and website redesign for RChain. One of it's tasks will be a full rebranding of the website, but in the meantime I plan to release "patches" (for lack of a better name) to the website on a weekly basis.

### Scope

Update the website. 
_Specifically this week: **I would like to update the homepage copywriting / messaging**_

- We'll stay on Squarespace
- We won't touch the visuals
- We won't fully reorganise the website

### DISCLAIMER

The goal is to roll out iterative improvements to the website on a weekly basis. Similar to how the bounty financial system got iteratively improved/patched by @lapin7 every month.

> Our target is not to get it perfect, but **to ship something** that's better than what we had before

We don't expect it to be perfect the 1st time. 
That's the whole point.

### How to get involved...

1. You can get involved by leaving feedback or input on the week's scope. I'll post a link to a draft page (once I've created one), so people can review the work-in-progress as it stands

2. We plan to hit "publish" this Friday at the latest. The next week, we'll do a 2nd update (and so on, every week)

3. I plan to post a schedule of roughly the next iterations, so everyone knows what to expect sometime this week

----

**Measure of completion:** website homepage update released with improved messaging / copy 
**Timeline:** this friday, or ideally even earlier
**Suggested budget:** $2.000

NOTE: @ian-bloom will act as an editor, and roll out releases. 
**[Here's a note I wrote to the branding WG, for more background](https://docs.google.com/document/d/11eBwL1kJY-wkMTcs8B9jm2yH-ntmkVO4CrE8Jw7KqR8/edit?usp=sharing)**

**Comment by dckc** [comment]
The web site is already released, so as stated, this is done.

I look forward to a refined title that says what the patch is about.

**Comment by pmoorman** [comment]
> I look forward to a refined title that says what the patch is about.

Good point. Done!

**Comment by dckc** [comment]
> Messaging is also hopelessly outdated.

That's not obvious to me. At a glance, I don't see any out of date messaging.

One glaring problem I see is: the **Events** link takes us to a page that doesn't mention the RChain Developer Conference April 14-18, 2018 in Boulder, CO noted on https://developer.rchain.coop/  - and in recent tweets... Most every tweet should link to a page in the **rchain.coop** domain. Let's quit sharecropping!

As I mentioned in #386:
I want the governance forum to have its own page; the current `/events` URL suggests that when the next event comes along, all the stuff about this event is going to get deleted

**Comment by ian-bloom** [comment]
@patrick727 Please contact me ASAP regarding adding Squarespace contributors and permission assignments (Site Owner,  Administrator,  Content Editor, etc.).

**Comment by dckc** [comment]
Hm. That's progress, but now what's the path from the home page to the gov forum?

I don't recommend using https://www.rchain.coop/events to stand for "2018 April Dev-Retreat" any more than for the gov forum. Each event should have its own page and `/events` should link to them all, highlighting the upcoming ones.

The dev retreat (conference, rather; consistency matters!) sort of has its own page, though it's PDF: https://developer.rchain.coop/rchain-developer-conference.pdf . You didn't choose that one, but a more forward-looking URL would have 2018 in it somewhere. See [Cool URIs don't change](https://www.w3.org/Provider/Style/URI).

The `old` in https://www.rchain.coop/events-old-2018-gov-forum suggests it's not going to be part of the web site going forward. I don't see any reason why not. In fact, we should link all the presentation materials and videos from there, directly or perhaps indirectly via https://www.gov.rchain.coop/ , since most of the gov forum scribes used google docs and porting to squarespace would just be a hassle.

Also, "Use the links below for details" is clutter; [Avoid talking about mechanics](https://www.w3.org/Provider/Style/NoMechanics.html).

**Comment by pmoorman** [comment]
@dckc good points regarding the event page(s). I'll add those to my list of things to update for a future patch. In fact, I'll create a Github issue right away to incorporate it.

---

### Current patch

Here's a WIP for this patch, that I hope to get released as soon as possible: https://www.rchain.coop/home-patch-1

**My aim has been to:**

- **Provide a plain english** "RChain for beginners" kind of description, that forms a good entrypoint to dive deeper into the world of RChain.

- **Improve the header** to not have parallax text scroll over other text or the logo

- **Put in a real headline,** with a real claim regarding scalable+reliable

- **Make all text better readable** (white background, bigger serif fonts). If it's more readable, more people will actually read it.
- **Insert a problem statement,** to _set a context_ for non-insiders to understand what we're setting out to achieve. 
- **Always talking in plain english,** rather than insider-english. Add a bit of an emotional touch where possible. 
- **Not make bold claims** without clearly outlining I) why they matter and II) backing them up with an explanation
- **Introduce Rhocalc as the solution,** before introducing RChain. This way it's more about "look what cool stuff we found" and less about "buy our thing"
- **Break down the technology into a 'bullet point' list of sorts,** that has for every developer something to get excited about. This is also a good setup to link to product pages with more info about each thing in the future.

**DISCLAIMER:** my aim is not to get this perfect, but rather to get it shipped as soon as possible. In the next couple hours I plan to tidy up the missing pieces in the technology section, and then it should be good to go.

Any improvements / suggestions / feedback can be incorporated at a later point, but shouldn't stop us from publishing (assuming we think this page is already better than the existing on) 



---

### Looking forward

To make this a little bit forward-looking, here's the next couple of patches that I have in mind:

**Other patches:**
1. Core messaging   _(that's this one)_
2. Team update  ::  owned by @ddayan
3. Information architecture
4. Call to Action area
5. Flesh out product pages
    1. Rho-calculus, Rholang & concurrency
    2. Correctness & reliability
    3. Casper POS  ::  owned by @cryptomillennial
    4. Namespaces
    5. Roadmap
6. Flesh out governance pages
    1. The coop
    2. About us / team
    3. Partners
    4. Events
7. Core messaging, v2.1  (incorporating new insights)
8. Visual style & identity

Things could change, obviously. My hope is to get #1 live any day now, and #2 and #3 next week for instance. We need to decide on #3 before we can publish #5, but #4 is not a blocker.

The event pages would fall under patch #3, for instance. 

---

### Collaborators

Finally, this is a good place to mention that I have 2 collaborators onboard who would also like to contribute to these patches. 

@ddayan (not his Github username, apparently) will help restructure the team/about section, and @cryptomillennial (also not Github username) has volunteered for at least writing for the Caspar page.

I will oversee that everything actually gets in Squarespace (because I have access now), and @ian-bloom will have "release power" so to speak  ;)

**Comment by ian-bloom** [comment]
@pmoorman - Where Pyrofex is mentioned, please include their logo.

**Comment by pmoorman** [comment]
@ian-bloom yes, noted!  

I won't do this in this patch, but instead make it part of the 2nd one (see here: https://github.com/rchain/bounties/issues/538).

**Comment by dckc** [comment]
wait.. what? why not do this patch?

**Comment by pmoorman** [comment]
In an effort to "small-scoped patches, but often" I want to push this to the 2nd thing. First thing is a clean focus on messaging (1st half of the page), next one about the 2nd half of the page.

@ddayan is working on this 2nd patch right now, and rethinks that entire section. Shouldn't take long anyway, but I'm keen to keep the scope of each patch focused & clean.

**Comment by dckc** [comment]
Ah. I misread; I thought you were punting on the whole 1st patch and rolling it into the 2nd.

**Comment by dckc** [comment]
Got a new ETA?

I see the original one was March 23. What do we know now that we didn't know when we made that guestimate?

**Comment by dckc** [comment]
@pmoorman and I were chatting and he asked me for a technical review. I'n **not OK** shipping the current copy:

### "Current blockchain platforms are broken in two ways"

It's so emphatic as to be incorrect. It's like saying the original SSL was broken. By today's standards, it is. But given that RChain isn't deployed yet, it's just as broken; for all we know, Ethereum's scalability issues will get fixed before RChain gets widely deployed.

perhaps: "Current blockchain platforms are holding back dApps in two ways: ..."

### "Even though platforms might be secure, the contracts running on them might not be."

RChain doesn't fix that either.

the way: the main feature that lets dApp builders formally verify their contracts is behavioral types. hard to say whether saying so in so many words is clear to our audience... but it's fair game as a bold claim that we can back up for the interested reader.


### "The Rhocalc execution model is both fundamentally concurrent and formally verified."

> RChain as a third generation blockchain platform: formally verified, fast, and scalable -- https://medium.com/rchain-cooperative/a-visualization-for-the-future-of-blockchain-consensus-b6710b2f50d6

what's formally verified is RChain, not Rho-calculus.

putting "formally verified" in the Rho-calculus section is the square peg in the round hole. I doubt I'll find a way to fit it in.

perhaps... "Rho-calcuclus is the fundamental model that lets us formally verify RChain"

### "users validating ... namespaces"

> ... shine when you have 100,000,000 users validating hundreds of namespaces ...

hmm... do users validate namespaces? I think in some sense operators of validators are users, but I'm getting the impression that typical users are not operators of validators. Which disappoints me, actually. I'm not quite sure about this. It would be good to hear from @nashef or @pyrocto on that.

**Comment by pyrocto** [comment]
In order for a validator to do validation, it has to get hold of the traffic; the only way to get that is to become a bonded validator in the region that interests you.  Users in the sense of clients that deploy new code aren't necessarily the same users running validators.

**Comment by pmoorman** [comment]
@dckc updated the comments you had, and also a few from Kent.

Here's the link again: https://rchain.coop/draft-home-patch-1

**Main updates:**

- "Current blockchain platforms are broken in two ways"  =>  "Current blockchain platforms are **holding back developers** in two ways:"

- "Even though platforms might be secure, the contracts running on them might not be."  =>  **Removed**. It got too complicated to replace it, and instead with just removing it we'll be fine.

- "The Rhocalc execution model is both fundamentally concurrent and formally verified."  =>  "The Rho-calculus execution model is fundamentally concurrent and naturally **suited for writing software that can be formally verified.**"

- "users validating ... namespaces"  =>  "RChain's real benefits shine when you have 100,000,000 users across hundreds of namespaces, all interacting seamlessly..."   (removed the mix-up between user/validator. I want to retain the notion on _'RChain benefits shine at really big scale'_)


Also addressed some other editorial things, and more such things, but it gets cumbersome to write that all down here. 

@dckc if you can (re)check the corrected sentences above, then I think we can push this live. Again: further editorial adjustments can be made on a rolling basis.

---

(cc @ian-bloom)

**Comment by dckc** [comment]
Looks good to me.

**Comment by PatrickM727** [comment]
we ought to word it in a way that leaves out "=>" 

besides that looks good

**Comment by dckc** [comment]
@pmoorman what's the next step here? Who has the ball?

**Comment by pmoorman** [comment]
Closing this (and it's sibling #538), since rollout turned out to be very hard, and the marketing organisation has decided to completely overhaul the website instead.
